### DISCUSSION TITLE
Make Global Filters Persist on Popular/All tabs

### DISCUSSION THREAD

**Discussion by sezonis** [Docile-Alligator/Infinity-For-Reddit#1163] [Ideas]
Global Filters are amazing, but would be way better if they could persist on the specific tabs "popular" and "all" because there are some subreddits that show disgusting (not NSFW, but just some stupid things like popping pimples and so on) content and also lots of useless content. By letting the global filters apply to these tabs, you have much more control of what you want to see on there. Making for a better viewing experience

**Comment by Docile-Alligator** [comment]
Just apply the filter to subreddit type "popular" and "all".

**Comment by greggh** [comment]
> Just apply the filter to subreddit type "popular" and "all".

How? I don't see that option anywhere?
